packages/scene-composer/src/SceneViewer.tsx

```diff
@@ -43,7 +43,10 @@ export const SceneViewer: React.FC<SceneViewerProps> = ({ sceneComposerId, confi
       return;
     }
 
-    const nodeRefs = composerApis.findSceneNodeRefBy(props.selectedDataBinding || '', [KnownComponentType.Tag]);
+    const nodeRefs = composerApis.findSceneNodeRefBy(props.selectedDataBinding || '', [
+      KnownComponentType.Tag,
+      KnownComponentType.DataBinding,
+    ]);
```

Comment:
this will be more likely to return multiple refs now. maybe you could check if the `selectedDataBinding` is an entity binding (e.g. only have `entityId`), if so, filter by `KnownComponentType.DataBinding`. otherwise filter by KnownComponentType.Tag.

Reply:
also, we should rename `KnownComponentType.DataBinding` to `KnownComponentType.EntityBinding` before release

Reply:
Add  test for if componentName is present and then added unit test to cover new option.  Will handle the data type renaming in future PR.